src/main/java/jenkins/scm/api/SCMSource.java

```diff
@@ -705,12 +705,12 @@ protected SCMRevision retrieve(@NonNull final String thingName, @NonNull TaskLis
      * @return a possibly empty set of revision names suggested by the implementation
      * @throws IOException if an error occurs while performing the operation.
      * @throws InterruptedException if any thread has interrupted the current thread.
-     * @since 2.5
+     * @since 2.6.0
      */
     @NonNull
     public final Set<String> fetchRevisions(@CheckForNull TaskListener listener, @CheckForNull Item context)
             throws IOException, InterruptedException {
-        return retrieveRevisions(defaultListener(listener));
+        return retrieveRevisions(defaultListener(listener), context);
```

Comment:
Not as bad as it sounds since the only proposed call site is [currently commented out anyway](https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/8dae8d968b2ec8ea727c83518280c6e0ee3d51fd/src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryConfiguration.java#L184-L192).